contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -711,8 +711,8 @@ CREATE OR REPLACE VIEW sys.spt_tablecollations_view AS
         o.object_id         AS object_id,
         o.schema_id         AS schema_id,
         c.column_id         AS colid,
-        CASE WHEN p.attoptions[1] LIKE 'bbf_original_name=%' THEN split_part(p.attoptions[1], '=', 2)
-            ELSE c.name END AS name,
+        CASE WHEN p.attoptions[1] collate "C" LIKE 'bbf_original_name=%' THEN split_part(p.attoptions[1] collate "C", '=', 2)
```

Comment:
In this case, split_part() works with deterministic collations only. So we need to ensure that we do not pass any column/var which is having non-deterministic collations. In this case, collation of p.attoptions[1] is already "C" so we don't need to specify collate "C" clause (I will remove this in next revision). 

But for other cases, we might need to specify the collation to ensure that there won't be any collation conflict. For example, equal operator, it will throw an error if both operands are having **non-default** collation and both are **different**. Same for **LIKE**. We also need to take care of such conflict in **UNION ALL** operation.